packages/block-library/src/image/image-editing/use-transform-image.js

```diff
@@ -12,6 +13,12 @@ function useTransformState( { url, naturalWidth, naturalHeight } ) {
 	const [ aspect, setAspect ] = useState();
 	const [ defaultAspect, setDefaultAspect ] = useState();
 
+	const imageCrossOrigin = useSelect( ( select ) => {
+		const value = select( 'core/block-editor' ).getSettings()
+			.imageCrossOrigin;
+		return typeof value === 'string' ? value : undefined;
```

Comment:
I wonder if we should avoid getting anything other than allowed values 🤔 

The [spec](https://html.spec.whatwg.org/multipage/urls-and-fetching.html#cors-settings-attribute) describes valid values.

Reply:
Good point.